client/rest/src/main/java/org/elasticsearch/client/RestClient.java

```diff
@@ -144,16 +145,16 @@ public Response performRequest(String method, String endpoint, Header... headers
      * @throws ClientProtocolException in case of an http protocol error
      * @throws ResponseException in case elasticsearch responded with a status code that indicated an error
      */
-    public Response performRequest(String method, String endpoint, Map<String, String> params, Header... headers) throws IOException {
-        return performRequest(method, endpoint, params, null, headers);
+    public Response performRequest(String method, String endpoint, Map<String, String> params, Header... headers) throws Exception {
+        return performRequest(method, endpoint, params, (HttpEntity)null, headers);
     }
 
     /**
-     * Sends a request to the elasticsearch cluster that the current client points to.
-     * Selects a host out of the provided ones in a round-robin fashion. Failing hosts are marked dead and retried after a certain
-     * amount of time (minimum 1 minute, maximum 30 minutes), depending on how many times they previously failed (the more failures,
-     * the later they will be retried). In case of failures all of the alive nodes (or dead nodes that deserve a retry) are retried
-     * till one responds or none of them does, in which case an {@link IOException} will be thrown.
+     * Sends a request to the elasticsearch cluster that the current client points to. Blocks till the request is completed and returns
```

Comment:
It probably doesn't matter. In the time I spent looking around to answer your question I found out that it has sort of graduated from slang into normal speech.